config/serverless.security.yml

```diff
@@ -35,3 +35,7 @@ xpack.fleet.internal.registry.spec.max: '3.0'
 # Serverless security specific options
 xpack.securitySolution.enableExperimental:
   - discoverInTimeline
+
+xpack.ml.ad.enabled: true
```

Comment:
The memory usage page is currently disabled in serverless.
This may be a mistake. I've pushed a [fix](https://github.com/elastic/kibana/pull/166651/commits/3f7315d561d98071790a258c490566d08e870dc3) to another [WIP PR](https://github.com/elastic/kibana/pull/166651) to change this. As that PR has some similar client side changes.